GATEWAY/src/pages/virtual-tiles/virtual-tiles.ts

```diff
@@ -119,6 +121,44 @@ export class VirtualTilesPage {
   }
 
   /**
+   * Opens a actionSheet showing application info
+   * Now only allows to start/stop the application
+   * Also displays a toast when completed.
+   */
+  public showApplicationInfo = () => {
+    let actionSheet = this.actionSheetCtrl.create({
+        buttons: [
+          {
+            handler: () => {
+              // Calls function to start/stop application
+              this.toggleAppOnline();
+
+              // Display toast verifying action is completed
+              let toastApplicationMsg = this.activeApp.appOnline ? 'Stopped runnning application ' : 'Started runnning application ';
+              toastApplicationMsg += this.applicationTitle;
+              let toast = this.toastCtrl.create({
+                duration: 3000,
+                message: toastApplicationMsg,
+              });
+              toast.present();
+            },
+            role: 'destructive',
+            text: this.appOnlineStatusMsg,
```

Comment:
if this is the only place this is used why don't you just remove the variable and use 
`this.activeApp.appOnline ? 'Stop application' : 'Start application';` instead?